src/sum-of-odd.js

```diff
@@ -25,7 +25,10 @@
  * @return {number}
  */
 function rowSumOddNumbers(row) {
-  // write code here
+  if (row < 1) {
+    return 0;
+  }
+  return Math.pow(row, 3);
```

Comment:
you can write like this
```suggestion
  return row < 1 ? 0 : Math.pow(row, 3);
```